fedbiomed/researcher/cli.py

```diff
@@ -89,7 +89,7 @@ class ConfigNameActionResearcher(ConfigNameAction):
             _this = self
             _component = ComponentType.RESEARCHER
 
-            def import_environ(self) -> 'fedbiomed.researcher.environ.Environ':
+            def set_component(self, config_name: str) -> 'fedbiomed.researcher.environ.Environ':
```

Comment:
@srcansiz here shouldn't we have something similar to `ConfigNameActionNode` ? eg:

```
            def set_component(self, config_name: str) -> None:
                config = ResearcherConfig(name=config_name)
```

(did not test if this enough, but some change is needed including for removing the dependency to `fedbiomed.researcher.environ`